packages/picasso-tailwind/src/index.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
     },
     borderWidth: {
       DEFAULT: '1px',
+      0: '0px',
```

Comment:
NIT: 0 doesn't need an unit, this is a funny line BTW, defining 0 to 0 :joy: 
```suggestion
      0: '0',
```

Reply:
Copied from the original TW config, just to support consistency

Reply:
We usually use '0px' in the config. Either always use "0" or "0px".

cc @TomasSlama

Reply:
I don't have a strong opinion about this, but I usually also copy from the Tailwind config. Why think about things that smart engineers already solved ;-)

Reply:
Let's keep values equal to default config.

Reply:
It's not a big deal, this is why I marked it as an `NIT`. It's up to you

> Why think about things that smart engineers already solved ;-)

We are also very smart engineers :grin: 
Just because someone wrote some code in that project that way doesn't mean that we should copy and paste it exactly, specially when the two projects have different coding styles, and different priorities.

I mentioned this change, because it is actually one of the [default rules enforced on stylelint](https://stylelint.io/user-guide/rules/length-zero-no-unit). And a good practice when writing CSS